Makefile

```diff
@@ -0,0 +1,9 @@
+VIM_COMPILER:=vim -N --cmd 'set rtp+=$$PWD' -c q -u
```

Comment:
Sure.  But we could also use make's `$(CURDIR)`?!
Just `.` seems nice and simple though.